.gitignore

```diff
@@ -32,5 +33,41 @@ scripts/coverage
 
 # typescript
 packages/*/*.tsbuildinfo
+apps/*/*.tsbuildinfo
 
+# Logs
 *storybook.log
+
+# Linux system files
+*~
+
```

Comment:
Ignore VScode settings
```suggestion
# editor
.vscode
```

Reply:
I think we want to include the `.vscode` settings it's something that @garrettbear added to add some color to our repo.